integration_tests/models/plugins/sqlserver.yml

```diff
@@ -45,6 +45,17 @@ sources:
           #     expression: "substr(split_part(metadata$filename, 'section=', 2), 1, 1)"
```

Comment:
let's remove this, I imagine it's from copy-pasting the snowflake example but possibly confusing to azure-only folks